Create ui-c8y-1021-0-0-acivating-user-switch-reworked.md (#2575)
Cumulocity-IoT/c8y-docs: content/change-logs/device-management/ui-c8y-1021-0-0-acivating-user-switch-reworked.md

---
date: ""

> title: Rework user switch activation

Reviewer: ```suggestion
title: Activating user in the Device credentials tab fixed
```

> title: Acivating user switch reworked.

Author: The Claude AI suggested to use the following title:
```suggestion
title: Rework user switch activation
```

> title: Activating user in the Device credentials tab

Reviewer 2: ```suggestion
title: Activating or deactivating users in the Device credentials page works reliably
```

> title: Activating user in the Device credentials tab fixed

Reviewer: ```suggestion
title: Activating user in the Device credentials tab
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-3875
version: 1021.0.0
---

> Acivating user switch reworked.

Author: The Claude AI suggested to use the following body:
```suggestion
To simplify the user experience when switching between users, the user switch activation has been reworked. Previously, the user had to manually activate the user switch in a separate step after selecting the user to switch to. Now, the user switch is automatically activated when selecting the user from the user switch dropdown menu. This change simplifies the user switching process and makes it more intuitive for users.
```

> Fixed issue in device credentials functionality allowing reliable user activation and deactivation.

Reviewer 2: ```suggestion
In the **Device credentials** page, user activation did not work properly. This issue has been fixed an activating or deactivating users now works reliably.
```

> To simplify the user experience when switching between users, the user switch activation has been reworked. Previously, the user had to manually activate the user switch in a separate step after selecting the user to switch to. Now, the user switch is automatically activated when selecting the user from the user switch dropdown menu. This change simplifies the user switching process and makes it more intuitive for users.

Reviewer: ```suggestion
Fixed issue in device credentials functionality allowing reliable user activation and deactivation.
```
